model/generate.go

```diff
@@ -159,7 +159,7 @@ func (g *GeneratedProject) NewVersion() (*Project, *ParserProject, *Version, *ta
 
 func (g *GeneratedProject) Save(ctx context.Context, p *Project, pp *ParserProject, v *Version, t *task.Task, pm *projectMaps) error {
 	if err := updateVersionAndParserProject(v, pp); err != nil {
-		return errors.Wrapf(err, "error saving version/parser project")
+		return err
```

Comment:
why, always use errors.Cause() to unwrap errors later, though I think that the db.IsDuplicateKey() error checker should unwrap if it doesn't already.

Reply:
I chose to unwrap this because the error returned from updateVersionAndParserProject is way more useful, and this is just vague.

Reply:
it's just safer to unwrap when using the error value and trust that we won't accidentally rewrap